builder.go

```diff
@@ -296,6 +325,15 @@ func (b *Builder) Persist(w io.Writer) error {
 		if err != nil {
 			return err
 		}
+		byts = []byte(n.conf)
```

Comment:
I'm guessing this isn't needed and is an artifact from earlier code; I'll check it out after merge.